CONTRIBUTING.md

```diff
@@ -1,34 +1,44 @@
 # Contributing OpenELA Documentation
 
 We welcome your contributions! There are multiple ways to contribute.
-All contributions to this repository and to OpenELA should following
-Contribution Guide including our Code of Conduct, which are included
-by reference here:
-https://github.com/openela/governance/blob/main/contributors_guidelines.md
 
-## Did you find a bug?
+All contributions to this repository and to OpenELA in general follow
+[the OpenELA Code of Conduct](https://github.com/openela/governance/blob/main/code_of_conduct.md).
+
+
+## Did you find an issue?
+
+Issues, be it a defect, a missing documentation part or any enhancement request, please [open an issue on GitHub](https://github.com/openela/openela-documentation/issues). Please provide as much detail as you can
```

Comment:
Possibly an odd line break here, but probably not serious.

Reply:
oh, thanks for noticing. I've fixed this.